msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java

```diff
@@ -42,12 +42,17 @@
     private final String mTenantId;
     private final AccessTokenRecord mAccessToken;
     private final IAccount mAccount;
+    private boolean mServicedFromCache;
 
-    AuthenticationResult(@NonNull final List<ICacheRecord> cacheRecords) {
+    AuthenticationResult(
+            @NonNull final List<ICacheRecord> cacheRecords,
+            final boolean isServicedFromCache
+    ) {
```

Comment:
As a formatting nit: could bring these braces up to end of the previous line

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1048/commits/0f34d4cb4448efedf0ee4b691126cad86662a7a9